src/components/calcite-label/calcite-label.scss

```diff
@@ -1,95 +1,56 @@
 :host-context([theme="dark"]) {
   @include calcite-theme-dark();
 }
-
 :host([dir="rtl"]) {
   text-align: right;
 }
-
 :host([scale="s"]) {
   --calcite-label-margin-bottom: 12px;
+  label {
+    @apply text--2;
+  }
 }
 :host([scale="m"]) {
   --calcite-label-margin-bottom: 14px;
+  label {
+    @apply text--1;
+  }
 }
 :host([scale="l"]) {
   --calcite-label-margin-bottom: 18px;
+  label {
+    @apply text-1;
+  }
 }
 
 :host label {
   cursor: pointer;
   width: 100%;
-  margin: 0 0 var(--calcite-label-margin-bottom) 0;
+  margin: 0 0 var(--calcite-label-margin-bottom, $baseline) 0;
   line-height: 1.3;
 }
 
-:host([scale="s"]) label {
-  @apply text--2;
-}
-
-:host([scale="m"]) label {
-  @apply text--1;
-}
-
-:host([scale="l"]) label {
-  @apply text-1;
-}
-
 :host([layout="default"]) {
   label {
     display: flex;
     flex-direction: column;
-    margin: 0 0 var(--calcite-label-margin-bottom, $baseline) 0;
-  }
-  & > ::slotted(.calcite-label-text) {
-    margin-bottom: 4px;
+    gap: 4px;
```

Comment:
☝️ this is (sorta) a breaking change, but only for browsers that don't support flex gap (IE11), but the workaround is very simple.  See the updated readme instructions for calcite-label on this PR for more details.